extensions/admin_ui/media/javascript/ui/panel/WelcomeTab.js

```diff
@@ -7,12 +7,7 @@
 WelcomeTab = function() {
 
   <%
-    @configuration = BeEF::Core::Configuration.instance
-    beef_proto = @configuration.get("beef.http.https.enable") == true ? "https" : "http";
-    beef_host = @configuration.get("beef.http.public") || @configuration.get("beef.http.host")
-    beef_port = @configuration.get("beef.http.public_port") || @configuration.get("beef.http.port")
-    beef_hook = @configuration.get("beef.http.hook_file")
-    hook_url = "#{beef_proto}://#{beef_host}:#{beef_port}/#{beef_hook}"
+    hook_url = BeEF::Core::Configuration.instance.hook_url
```

Comment:
This isn't a one-for-one replacement as `hook_url` doesn't include the hook path portion of the URL. I'm not sure if that matters.